espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Router.java

```diff
@@ -126,16 +127,24 @@
         routeMap.addRoute(new RouteDefinition(Method.GET, "/session/:sessionId/element/:elementId/size", new GetSize(), AppiumParams.class));
         routeMap.addRoute(new RouteDefinition(Method.GET, "/session/:sessionId/element/:elementId/text", new Text(), AppiumParams.class));
         routeMap.addRoute(new RouteDefinition(Method.POST, "/session/:sessionId/element/:elementId/value", new SendKeys(), TextParams.class));
+        routeMap.addRoute(new RouteDefinition(Method.GET, "/session/:sessionId/window/:windowHandle/size", new GetWindowSize(), AppiumParams.class));
```

Comment:
is windowHandle simply ignored?

Reply:
@mykola-mokhnach Yeah, because it's only ever one window (the current one). I'd like to leave it there though in case we ever automate multiple windows.